{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/utils/emails.py

```diff
@@ -37,34 +44,19 @@ def send_html_email(subject, template, send_from, send_to, context={}, bcc_email
     #         if domain not in valid_domains:
     #             return
 
-    assert isinstance(send_to, (list, tuple, str)), "send_to must be an instance of list, tuple, or str"
+    if not isinstance(send_to, (list, tuple, str)):
+        raise Exception("send_to must be an instance of list, tuple, or str")
```

Comment:
You shouldn't really use `assert` in real code as it doesn't do proper checking. It should be used in tests only.